### PR TITLE
Upgrading for the preview release SDK

### PR DESCRIPTION
**Pull request by phillipov**
While it has not been exhaustively tested, it will build with the preview Gaia SDK. After merging this, we can work on converting this into a sandbox-based app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,4 +43,12 @@ Now when you go back to the UI in your browser it should look like this and it c
...
-If you have any questions or feedback please reach out to us at Gaia at info@gaiaplatform.io or directly to the author of this example at steve@gaiaplatform.io
+If you changed the ruleset or the C++ code, rebuild it.
+```
+cd build
+make -j$(nproc)
+```
+
+If you changed the schema DDL, run `./rebuild.sh`. This will erase the data in the database, but it's necessary to sync the schema with the state of the database.
+
+If you have any questions or feedback please reach out to us at Gaia at [info@gaiaplatform.io](mailto:info@gaiaplatform.io) or directly to the author of this example at [steve@gaiaplatform.io](mailto:steve@gaiaplatform.io).
```
Comment (stevegaia) [inline README.md:54]: Let's go ahead and take my name out. I think info@gaiaplatform.io *should* be sufficient.

**Review by kenneth-gaia — APPROVED** [review]
I don't see anything glaringly obvious

**Review by stevegaia — COMMENTED** [review]
Let's add one more reviewer from outside apps. Like Laurentiu or Simone. Since both this version and the sandbox version will be public facing I'd like one more set of eyes on preferred Gaia coding style.

File: README.md (hunk 1 above)
Comment (phillipov) [reply README.md:54]: "Cleaning up" the ruleset should happen after we've sandboxified the app. Waiting for more eyes would be a speed bump and reduce the time that Kenneth could help out.

I will remove your email, good idea.

**Comment by stevegaia** [comment]
> While it has not been exhaustively tested, it will build with the preview Gaia SDK. After merging this, we can work on converting this into a sandbox-based app.

Thinking about this some more, I like the idea of both the standalone version that talks to the ui through named pipes (I.e. - this version) and the version that talks to the sandbox. So let's add that change we talked about this morning to pull down the fontawesome js file so that others don't bump into the localhost vs. IP address issue.

**Review by stevegaia — APPROVED** [review]

**Pull request merged**